File: VB/WindowsFormsApplication9/CustomEdit1PopupForm.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraEditors.Popup

Namespace WindowsFormsApplication9
    Public Class CustomEdit1PopupForm
        Inherits PopupDateEditForm

        Public Sub New(ByVal ownerEdit As CustomEdit1)
            MyBase.New(ownerEdit)
        End Sub

        Protected Overrides Function CreateViewInfo() As PopupBaseFormViewInfo
            Return MyBase.CreateViewInfo()
        End Function
        Protected Overrides Function CreateCalendar() As DevExpress.XtraEditors.Controls.DateEditCalendar
            Return New CustomDateEditCalendar(OwnerEdit.Properties, Me.OwnerEdit.EditValue)
        End Function
    End Class
End Namespace

File: CS/WindowsFormsApplication9/CustomEdit1PopupForm.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraEditors.Popup;

namespace WindowsFormsApplication9 {
    public class CustomEdit1PopupForm : PopupDateEditForm {
        public CustomEdit1PopupForm(CustomEdit1 ownerEdit)
            : base(ownerEdit) {
        }

        protected override PopupBaseFormViewInfo CreateViewInfo() {
            return base.CreateViewInfo();
        }
        protected override DevExpress.XtraEditors.Controls.DateEditCalendar CreateCalendar() {
            return new CustomDateEditCalendar(OwnerEdit.Properties, this.OwnerEdit.EditValue);
        }
    }
}
